packages/storybook/stories/open-forms/StoryUtil.tsx

```diff
@@ -53,6 +53,10 @@ type SingleFormioComponentProps = {
   extraComponentProperties?: { [index: string]: any };
 };
 
+type MultipleFormioComponentsProps = {
+  components?: any;
```

Comment:
Is dit dan niet van het type `SingleFormioComponentProps[]`?

Reply:
Ik zie dat je dan gedoe gaat krijgen met de extraComponentProperties. We zouden eventueel dit kunnen doen:

```
type SingleFormioComponentProps = {
  type: string;
  key: string;
  label: string;
  [index: string]: any;
};

type MultipleFormioComponentsProps = {
  components: SingleFormioComponentProps[];
};
```

Dan moeten de SingleFormioComponent stories wel gerefactored worden om geen `extraComponentProperties` meer te gebruiken.

Reply:
dan gaan er wel veel stories stuk die de extraComponent properties gerbuiken